images/PoweredbyJupyter-darkborder.svg

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="240px" height="89px" viewBox="0 0 240 89" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 40.3 (33839) - http://www.bohemiancoding.com/sketch -->
+    <title>PoweredbyJupyter-darkborder-2</title>
```

Comment:
we could actually put the dark one into the footer! (after removing the backgound to make sure it’s the exact same color)